public/locales/fr/common.json

```
@@ -1,47 +1,47 @@
 {
   "scrobble": "Scrobble",
   "getScrobbling": "Début du scrobbling",
-  "loading": "Chargement...",
+  "loading": "Chargement ...",
   "artist": "Artiste",
   "song": "Chanson",
   "title": "Titre",
   "album": "Album",
-  "scrobbleFromAlbum": "Scrobble albums",
-  "scrobbleAlbum": "Scrobble album",
+  "scrobbleFromAlbum": "Scrobbler des albums",
+  "scrobbleAlbum": "Scrobbler l'album",
   "album_plural": "Albums",
-  "goBack": "Go back",
-  "topAlbumsBy": "Albums by {{nameOfArtist}}",
-  "noAlbumsFound": "No albums found for <1>{{albumOrArtist}}</1>",
-  "noArtistsFound": "No artists found for <1>{{albumOrArtist}}</1>",
-  "emptyAlbum": "This album appears to be empty.",
-  "scrobbleSongs": "Scrobble songs",
-  "scrobbleSelected": "Scrobble selected",
-  "findAlbumCopy": "Enter an album or artist name",
-  "customTimestamp": "Custom timestamp",
-  "albumTimestampLogicDescription": "Scrobbling tracks \"now\" will scrobble the last track at the current time, backdating the previous ones accordingly (as if you had just finished listening). The \"custom timestamp\" will define the timestamp of the first track and add time to the following tracks from there (i.e. you specify the time you started listening to this album)",
+  "goBack": "Retour en arrière",
```

Can we use `Retourner` instead? (this "go back" is used in a context of navigation, as the browser back/forward arrows, for example)

```suggestion
  "goBack": "Retourner",
```

---

I prefer to say `Retour en arrière` or `Revenir à la page précédente` than to say `Retourner` but it's a personal choice

---

Let's go with `Retour` then, to keep it short

```suggestion
  "goBack": "Retour",
```